spanish-translations.md

```diff
@@ -0,0 +1,85 @@
+# Automatic Spanish Translations
+
+We're happy to announce that from now on we'll deliver Spanish translations to our new
+tutorials (scripts and subtitles). How did we make this possible? You might think that
+we hired a professional who takes care of translating the content. As great as human beings, 
+I'm proud to say that you would be wrong! As programmers, we love to automate processes, so, that's
+exactly what we did! We leverage the powerful translations service of *DeepL*, which is
+a translation service based on an AI algorithm. And... it's surprisingly good! But, 
+this time it wasn't as easy as making an API call, we had to heavily customize our 
+content so that some technical words or code blocks won't get translated and also keep
```

Comment:
```suggestion
content so that technical words and code blocks are *not* translated. We also need to help Deepl
understand some of our technical language.
```